### PR TITLE
Document current approach in main README

### PR DESCRIPTION
**Pull request by spier**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,33 @@
...
 ```sh
 rsync -avz --include "*/" --exclude "*" international/ <YOUR_COMMUNITY_NAME>
 find <YOUR_COMMUNITY_NAME>/ | xargs -I BASEPATH touch BASEPATH/.keep
 ```
+
+## Questions?
+
+This toolkit is really new!
+
+Please contact us in `#local-community` in [Slack](https://innersourcecommons-inviter.herokuapp.com) with any questions.
```
Comment (yuhattor) [inline README.md:33]:
```suggestion
Please contact us in `[#local-community](https://innersourcecommons.slack.com/archives/C046MD5R5RT)` in [Slack](https://innersourcecommons-inviter.herokuapp.com) with any questions.
```
Adding the link for #local-community channel

File: README.md [hunk 2]
```diff
@@ -1,16 +1,33 @@
...
+
+Luckily this toolkit by the [InnerSource Commons](https://innersourcecommons.org) is just what you need to get your InnerSource community started. :)
+
+## Existing communities
+
+* [Japan](./japan) - This is the first regional community that formed. Therefore much of the material in here was extracted from their work.
+  * communcation happens in Slack via `#jp-general` and `#jp-contents`
+  * with the tag "**Japan :jp:**" you can filter for their [issues](https://github.com/InnerSourceCommons/community-toolkit/labels/Japan%20%3Ajp%3A) and [discussions](https://github.com/InnerSourceCommons/community-toolkit/discussions?discussions_q=label%3A%22Japan+%3Ajp%3A%22)
+  * their [Public Community Roadmap](https://github.com/orgs/InnerSourceCommons/projects/1) shows their plans for the coming quarters
+* [Brazil](./brazil) - You find them in `#innersource-brazil`. Their directory in here was just created.
```
Comment (yuhattor) [inline README.md:14]:
```suggestion
* [Brazil](./brazil) - You find them in `[#innersource-brazil](https://innersourcecommons.slack.com/archives/C03JP108XGE)`. Their directory in here was just created.
```

File: README.md [hunk 3]
```diff
@@ -1,16 +1,33 @@
...
-Fortunately, InnerSource Commons has them. This toolkit is just what you need to get started in your InnerSource community.
+When you are starting an InnerSource Community, you will need a toolkit to get your community off the ground quickly.
+This is true no matter whether your community is a company-internal activity, a regional community, or a specific business-to-business sharing group.
+
+Luckily this toolkit by the [InnerSource Commons](https://innersourcecommons.org) is just what you need to get your InnerSource community started. :)
+
+## Existing communities
+
+* [Japan](./japan) - This is the first regional community that formed. Therefore much of the material in here was extracted from their work.
+  * communcation happens in Slack via `#jp-general` and `#jp-contents`
```
Comment (yuhattor) [inline README.md:11]:
```suggestion
  * communcation happens in Slack via `[#jp-general](https://innersourcecommons.slack.com/archives/C03M546NR16)`
```

**Review by yuhattor — APPROVED** [review]
Looks wonderful!
Surely we need a list of the existing slack channel:)

**Comment by spier** [comment]
There was a bug in the formatting of the slack channels, which I just fixed in [6526351](https://github.com/InnerSourceCommons/community-toolkit/commit/6526351092110d4d54800c98b04286eacb3b1180)

**Pull request merged**